tests/models/test_cli.py

```diff
@@ -40,50 +42,38 @@ def sk_model(iris_data):
     return knn_model
 
 
-def test_predict_with_old_mlflow_in_conda_and_with_orient_records(iris_data):
```

Comment:
btw, this is a pretty reasonable test post-1.0 to make sure we maintain the API. Should we plan to add it back?

Reply:
Yes that's a good point. We should put it back. Can put it back now and mark it as a skip and put a todo.